migrations/20180503104733_order_items.js

```diff
@@ -0,0 +1,13 @@
+exports.up = (knex, Promise) => {
+  return knex.schema.createTable('order_items', (table) => {
+    table.increments('id').primary()
+    table.int('user_id').references()
+    table.int('order_id').references()

```

Comment:
These 2 calls to `references()` should include a string to the associated `'table.primary_key'`.